### ISSUE TITLE
Interpunct (middot) not positioning properly

### ISSUE THREAD

**Issue by jorditarrida** [jpt/barlow#46]
Hi!
First of all, thanks for your work on this awesome font.
I am loading Barlow Regular from Google Fonts on a website. When using the interpunct (middot) character, which is a needed character in Catalan language, it is positioned incorrectly. This is how the word "col·laboradors" is rendered, notice the middot on top of the first l:
![Captura de Pantalla 2019-03-18 a les 16 53 37](https://user-images.githubusercontent.com/4738498/54544261-8286a280-499f-11e9-9ee4-2fd92280a0c5.png)
I am sure I am using the correct middot character (U+00B7). This is definitely not the expected behaviour, but I am not sure if this is a font problem, a Google Fonts problem, or a problem on my end. Could you please investigate this issue and let me know if there is anything I can do on my side to fix this?
Thanks in advance!
Jordi

**Comment by jpt** [comment]
Hi @jorditarrida, thank you for opening the issues, I know there have been problems with the middot before -- I'll take a look.

**Comment by jorditarrida** [comment]
Hi! Just a quick update, we found out that some browsers are rendering the middot correctly. In the following screenshot, left is Chrome (incorrect) and right is Safari (correct):

![Captura de Pantalla 2019-04-25 a les 13 57 34](https://user-images.githubusercontent.com/4738498/56734097-3f6bdc00-6762-11e9-99d2-12e4fb6b023b.png)

I've seen on other issues that you are working on a new version of the font which corrects some of these issues. Do you have an estimated release date for the new version?

Thanks in advance!
Jordi

**Comment by jdivins** [comment]
Same problem here!
Subscribed to the issue to check resolution. 
Thank you @jpt

**Comment by thatcort** [comment]
On my site, the middot is rendering on the baseline as a period:

<img width="101" alt="image" src="https://user-images.githubusercontent.com/12662578/59703667-637af500-91c8-11e9-8c12-773b4881e0f7.png">

**Comment by jpt** [comment]
Hi @thatcort, @jorditarrida,  @jdivins -- can you check the latest [version 1.422, which you can download here](https://github.com/jpt/barlow/archive/master.zip) (you'll find the files in the `fonts` folder). Seems OK to me know! I'll close the issue and and alert distributers (e.g. Google Fonts) if this fixes it.

**Comment by thatcort** [comment]
I just pulled and indeed the interpunct character is now at the correct height. Thank you!

**Comment by jpt** [comment]
Thanks for confirming @thatcort ! I'll close this issue now that it's fixed here. To everyone using Barlow from Google Fonts, putting the wheels into motion / about to open a PR with them now -- thank you for your patience

**Comment by orioltestart** [comment]
Hi, I've noticed that this still happens. I also saw that this happens when the document is on "ca" language (<html lang="ca">). It renders correctly on other languages.

Maybe it doesn't have Catalan language support?
Thanks

**Comment by jorditarrida** [comment]
@jpt I could check what @orioltestart noted and can confirm it's true, and even wrote a Codepen to show it: https://codepen.io/jorditarrida/pen/RwWYNLj

As seen, the issue is only present when the lang is `ca`, with any other real (`en`, `es`) or fake (`çç`) language code, the character renders properly. I set the `lang` attribute to different `div`s to test this issue.

I tested this issue in Google Chrome on macOS loading the font from Google Fonts. Here's a screenshot in case other browser/OS combinations don't have the issue:

![Captura de Pantalla 2020-05-14 a les 10 20 11](https://user-images.githubusercontent.com/4738498/81910738-9e4dfe80-95cc-11ea-92a7-d0071f6019f2.png)
